packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.test.js

```diff
@@ -691,4 +691,117 @@ describe('InputDateTimePicker', () => {
 			expect(onError).not.toHaveBeenCalled();
 		});
 	});
+
+	describe('dropdown management', () => {
+		it('should have the dropdown closed by default', () => {
+			const wrapper = shallow(<InputDateTimePicker />);
+
+			const overlayWrapper = wrapper.find('Overlay');
+			expect(overlayWrapper.prop('show')).toBe(false);
+		});
+
+		it('should open the dropdown on input focus', () => {
+			const wrapper = shallow(<InputDateTimePicker />);
+
+			const inputWrapper = wrapper.find('DebounceInput');
+
+			inputWrapper.prop('onFocus')();
```

Comment:
You mount it, shouldn't you simulate the focus on input ? (I mean instead of getting a props and calling it)

Reply:
Yes I can and I did it !! Even if under the hood, it's the same. Now it's more semantic +1. :p

I realized I have forgotten some cases with focus outside, so I added them too.